bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -245,6 +250,20 @@ public void disableForceGC() {
         }
     }
 
+    public void enableForceGCIfMinIntervalElapsed() {
+        long curTime = System.currentTimeMillis();
+        if ((lastMainCompactionTime.longValue() - curTime) > forceCompactionInterval) {
```

Comment:
Will `curTime` always bigger than `lastMainCompactionTime`?

Reply:
yeah curTime will be >= lastMain isn't it @reddycharan ?

Reply:
bummer..yes, it should be other way round. In next iteration I'm planning to add testcases.